backend/canvas_app_explorer/views.py

```diff
@@ -41,6 +41,13 @@ def list(self, request: Request) -> Response:
         serializer = serializers.LtiToolWithNavSerializer(
             queryset, many=True, context={ 'available_tools': available_tools }
         )
+
+        # loop through the available tools and add the sessionless launch URL
+        for tool in serializer.data:
```

Comment:
This probably will need to be moved into an API method for both performance and functionality.

We really don't need to make an API call against every tool every time.